DayrSuryan/DSEthiop23.xml

```diff
@@ -62,99 +62,102 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <locus from="1" to="24r"/>
                      <title type="complete" ref="LIT5911Heggi"/>
                      <textLang mainLang="ti"/>
-                     <note>The text is the collection of the statutes of the Coptic Church
-                        approved on <date>2 June 1985</date> (the feast of <persName ref="PRS4306GabraMa"/>). 
-                        The statutes were issued in Arabic and English; the translator, ʾabuna Maqāryos, used for his translation the Arabic version.
-                        The text contains 89 chapters or statutes (<foreign xml:lang="gez">ዓንቀጽ</foreign>). The last one concerns the order of 
-                        making changes in the statutes 
-                        (the procedure in stages, marked <foreign xml:lang="gez">ሀ</foreign>, 
+                     <note>The text is a collection of the Statutes of the Coptic Church
+                        approved on <date>2 June 1985</date> (the feast of <persName ref="PRS4306GabraMa"/>).
+                        The statutes were issued in Arabic and English; the translator, ʾabuna Maqāryos,
+                         used for his translation the Arabic version (he added English words to the Tigrinya text, to clarify some terms).
+                        The text contains 89 chapters or statutes (<foreign xml:lang="gez">ዓንቀጽ</foreign>). The last one concerns the order of
+                        making changes in the Statutes
+                        (the procedure in stages, marked <foreign xml:lang="gez">ሀ</foreign>,
                         <foreign xml:lang="gez">ለ</foreign>, <foreign xml:lang="gez">ሓ</foreign>, <foreign xml:lang="gez">መ</foreign>).  </note>
                      <incipit xml:lang="ti">
-                         በስመ፡  አብ፡  ወወልድ፡  ወመንፈስ፡  ቅዱስ፡   አሐዱ፡  አምላክ፡  አሜን ። 
-                         ዓንዲ፡  ሕጊ፡  ናይ፡  ግብጺ፡  ኦርቶዶክስ፡  ቤተ፡  ክርስቲያን፡  ሲኖዶስ። ንናይ፡  ግብጺ፡  እስክንድርያ፡  ኦርቶዶክስ፡  ቤተ፡  ክርስቲያንን፡  ናይ፡  
-                         ሓውርያ፡  ማርቆስ፡  ሃገረ፡  ስብከትን፡  ኵሉ፡  መማኀዳደሪ።.....
-                     ቀዳማይ፡  ክፍሊ፡  መእታዊ፡  (መቅድማት)
-                     ዓንቀጽ፡  (፩)
-                     ናይ፡  እስክንድርያ፡  ኦርቶዶክስ፡  ናይ፡  ግብጺ፡  ቤተ፡  ክርስቲያን፡  ሓዋርያዊት፡  ቤተ፡  ክርስቲያን፡  እያ፡  ርክባ፡  ከኣ፡  ምስተን፡  ናይ፡  ምብራቅ
-                     ፡  (oriental orthodox) ኦርቶዶክስ፡   ብእምነትን፡  <pb n="2r"/>ምሥጢራተ፡  ቤተ፡  ክርስቲያንን፡  ልማድን፡  (ecclesiastical tradition) ምሳኣ
-                     ፡  ርክብ (ኅብረት፡  ዘለወን፡  እይ     
+                         በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ አሜን ።
+                         ዓንዲ፡ ሕጊ፡ ናይ፡ ግብጺ፡ ኦርቶዶክስ፡ ቤተ፡ ክርስቲያን፡ ሲኖዶስ። ንናይ፡ ግብጺ፡ እስክንድርያ፡ ኦርቶዶክስ፡ ቤተ፡ ክርስቲያንን፡ ናይ፡
+                         ሓውርያ፡ ማርቆስ፡ ሃገረ፡ ስብከትን፡ ኵሉ፡ መማኀዳደሪ።...
+                     ቀዳማይ፡ ክፍሊ፡ መእታዊ፡ (መቅድማት)
+                     ዓንቀጽ፡ (፩)
+                     ናይ፡ እስክንድርያ፡ ኦርቶዶክስ፡ ናይ፡ ግብጺ፡ ቤተ፡ ክርስቲያን፡ ሓዋርያዊት፡ ቤተ፡ ክርስቲያን፡ እያ፡ ርክባ፡ ከኣ፡ ምስተን፡ ናይ፡ ምብራቅ፡
+                     (oriental orthodox) ኦርቶዶክስ፡  ብእምነትን፡ <pb n="2r"/>ምሥጢራተ፡ ቤተ፡ ክርስቲያንን፡ ልማድን፡ (ecclesiastical tradition) ምሳኣ፡
```

Comment:
Do you remember if the English in paratheses is written in the manuscript? I don't find the right image, but I think you have it?

Reply:
I have no images, but I cannot imagine that anyone of us could add this.

Reply:
No, not really...